### PR TITLE
[Inbound Calling] Calling stateful sample fixes

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Fixes issues 
- stable build was not happening
- Call manager wasn't updating correctly
# Why

- proper cc to best reflect the feature
- allows users to work with proper calls

# How Tested

Validated and built locally

### CODE REVIEW

File: samples/CallingStateful/src/App.tsx [hunk 1]
```diff
@@ -39,8 +50,30 @@ function App(): JSX.Element {
   /* @conditional-compile-remove(one-to-n-calling) */
   const [callAgent, setCallAgent] = useState<DeclarativeCallAgent | DeclarativeTeamsCallAgent>();
   const [call, setCall] = useState<Call | TeamsCall>();
+  /* @conditional-compile-remove(one-to-n-calling) */
   const [calls, setCalls] = useState<Call[] | TeamsCall[]>([]);
 
+  const statefulCallClientUpdatedListener = useCallback(
```
Comment (edwardlee-msft) [inline samples/CallingStateful/src/App.tsx:56]: Should this whole thing be CCed?

File: samples/CallingStateful/src/App.tsx [hunk 2]
```diff
@@ -39,8 +50,30 @@ function App(): JSX.Element {
...
+    (newStatefulCallClient: CallClientState): void => {
+      if (newStatefulCallClient.callAgent && /* @conditional-compile-remove(one-to-n-calling) */ callAgent?.calls) {
+        /* @conditional-compile-remove(one-to-n-calling) */
+        setCalls((callAgent?.calls as Call[] | TeamsCall[]) || []);
+      }
+    },
+    [/* @conditional-compile-remove(one-to-n-calling) */ callAgent]
+  );
+
```
Comment (edwardlee-msft) [inline samples/CallingStateful/src/App.tsx:65]: Should these be useEffects be CCed?

**Review by edwardlee-msft — APPROVED** [review]

File: samples/CallingStateful/src/components/CallManager.tsx [hunk 3]
```diff
@@ -67,7 +71,7 @@ const CallItem = (props: CallItemProps): JSX.Element => {
 
   useEffect(() => {
     if (call.state === 'Connecting') {
-      setCallTitle('Connecting');
+      setCallTitle('');
```
Comment (JamesBurnside) [inline samples/CallingStateful/src/components/CallManager.tsx:74]: Why this change?

**Review by JamesBurnside — APPROVED** [review]

File: samples/CallingStateful/src/components/CallManager.tsx (hunk 3 above)
Comment (dmceachernmsft) [reply samples/CallingStateful/src/components/CallManager.tsx:74]: I don't have designs for this component so just building the experience that feels right, so this text having the call state where the person's name should be felt weird so made it a empty placeholder until the call connects
